modules/@apostrophecms/doc/ui/apos/apps/AposDoc.js

```diff
@@ -29,10 +29,17 @@ export default () => {
     if (!modal) {
       throw new Error(`${type} is not a valid piece or page type, or cannot be edited`);
     }
+
+    const copyOfId = typeof copyOf === 'string' ? copyOf : copyOf?._id;
```

Comment:
Please update it to accept `copyOfId` directly and document that as the correct solution. Support for `copyOf` should only be a fallback when `copyOfId` is not present.